packaging/ios/build.sh

```diff
@@ -157,8 +157,7 @@ echo next building for $ARCH
 	fi
 
 	if [ ! -e libxml2-${LIBXML2_VERSION}.tar.gz ] ; then
-		# wget ftp://xmlsoft.org/libxml2/libxml2-${LIBXML2_VERSION}.tar.gz
-		wget http://api.github.com/repos/GNOME/libxml2/tarball/v${LIBXML2_VERSION} -O libxml2-${LIBXML2_VERSION}.tar.gz
+		curl -O ftp://xmlsoft.org/libxml2/libxml2-${LIBXML2_VERSION}.tar.gz
```

Comment:
I actually switched away from xmlsoft for this because their website was down for weeks - api.github.com occasionally times out, but has MUCH higher availability
But fundamentally it might make more sense to just check it out from git